### PR TITLE
Loadout tools will now save job title, id's access and iff signal, and headset

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Iff signal and access can be customised on the id before saving
Job (so the title, the rank, the skillset) and headset are hardcoded (if the path doesn't exist in the code, it won't save anything). So you can't make custom jobs

fixes https://github.com/tgstation/TerraGov-Marine-Corps/issues/7467

## Changelog
:cl:
expansion: Loadout tools will now save job title, id's access and iff signal, and headset
/:cl:

### CODE REVIEW

File: code/datums/loadout/item_representation/_item_representation.dm [hunk 1]
```diff
@@ -118,3 +118,32 @@
...
 	stack.update_icon()
 	return stack
+
+/**
+ * Allow to representate an id card (/obj/item/card/id)
+ */
+/datum/item_representation/id
+	/// the access of the id
+	var/list/access = list()
+	/// the iff signal registered on the id
```
Comment (TiviPlus) [inline code/datums/loadout/item_representation/_item_representation.dm:128]: Will this not create free clonable id cards for marines

Comment (BraveMole) [reply code/datums/loadout/item_representation/_item_representation.dm:128]: this is only for admins (through loadout tool). The id slot is not saved by the loadout vendors

Comment (BraveMole) [reply code/datums/loadout/item_representation/_item_representation.dm:128]: And even if you put an id in your bag, you can't find any in vendors. So the loadout vendor would not sell it back to you

Comment (TiviPlus) [reply code/datums/loadout/item_representation/_item_representation.dm:128]: ok

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**